rest/models/UserIdentity.go

```diff
@@ -29,6 +29,7 @@ type UserIdentity struct {
 	DashboardTemplates []DashboardTemplate               `json:"dashboardTemplates,omitempty"`
 	UIPreview          bool                              `json:"uiPreview"`
 	UIPreviewSeen      bool                              `json:"uiPreviewSeen"`
+	ActiveWorkspace    datatypes.JSONType[Workspace]     `json:"activeWorkspace"`
```

Comment:
I think this just has to be ID of the workspace. I think it should be a uuid so string for us

Reply:
Updated to use string type over JSON